scripts/variantstore/beta_docs/run-your-own-samples.md

```diff
@@ -105,26 +128,59 @@ If your data is not stored in the cloud, you’ll need to upload it to your work
 ---
 
 **Warning:**        
-The workflow in the GVS beta workspace is configured based on the format of the TSV file. To avoid reconfiguring the workflow, do **not** rearrange or rename the columns in the TSV file. If you need to customize the column names, read more about the parameters to use in [GVS Bulk Ingest Details](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/docs/gvs-bulk-ingest-details.md).
+The workflow in the GVS beta workspace assumes the table with samples is called `sample`. If this is not the case for your data, set the `samples_table_name` variable in the inputs to your table name as a string in the format "*TABLE_NAME*".
 
 ---
 4. Follow the steps in [How to use the Data Uploader](https://support.terra.bio/hc/en-us/articles/4419428208411) to **upload your data and TSV file** to Terra.
 
+### For Importing Data from TDR
+
+#### Delete the example data
+
+You will need to delete the example sample data from your workspace so that it is not included in your callset.
+
+1. Navigate to the **Data tab** in your clone of the GVS workspace.
+2. Click on the **sample** data table.
+3. Click the **top check mark** to select all samples.
+4. Click **Edit** and click **Delete selected rows** to delete the data.
+
+#### Export TDR Snapshot to the workspace
+
+1. Navigate to the data snapshot that needs to be processed. 
+2. Select the EXPORT SNAPSHOT tab once the snapshot is selected. The sample rows should include a `sample_id`, `gvcf_path`, and `gvcf_index_path` at a minimum.
+3. Make sure “Convert DRS URLs to Google Cloud Storage Paths (gs://)" is selected, then click the “EXPORT SNAPSHOT” button.
+   1. **Warning: The workflows will fail without selecting this.**
+4. Choose the workspace created in Step 7 of the Setup instructions in [gvs-quickstart](./gvs-quickstart.md)
+
+--- 
+
+**Warning:**        
+The workflow in the GVS beta workspace assumes the table with samples is called `sample`. If this is not the case for your data, set the `samples_table_name` variable in the inputs to your table name as a string in the format "*TABLE_NAME*".
+
+---
 ## Run the workflow
 
-Now that your samples are loaded into data table in Terra, it’s time to setup and run the GVS workflow!
+Now that your samples are loaded into data table in Terra, it’s time to setup and run the GVS workflow! 
+
+---
+**Notes**
+1. Note that GVS **expects reblocked gVCFs**. If your gVCFs are not reblocked, see the text above on how you can use the ReblockGVCF workflow in this workspace.
```

Comment:
Sometimes you've got gVCF and sometimes you've got GVCF and sometimes gvcf.  Geraldine preferred GVCF.  I don't know that I care, but consistency would be nice.  Maybe IntelliJ case sensitive find and replace?

Reply:
Done!